Neos.Flow/Classes/Mvc/Controller/ActionController.php

```diff
@@ -212,6 +212,10 @@ public function processRequest(ActionRequest $request, ActionResponse $response)
         }
 
         $this->callActionMethod();
+
+        if ($this->response->getContentType() === '') {
```

Comment:
This now allows any view or render method to set the content type to fix https://github.com/neos/flow-development-collection/pull/2005#issuecomment-707625891